<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2100</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>2041</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>P</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationC</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="aEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>申请A资源数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>申请B资源数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>申请C资源数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>1190</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复t0状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>1190</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>请求资源</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>2041</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>P</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>WorkA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>WorkB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>WorkC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NeedC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AllocationC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>W+A(A)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>W+A(B)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>W+A(C)</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="showLabel">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>1180</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel_2">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>1180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel_3">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>1180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel_4">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>1180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel_5">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>1180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>1120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>安全序列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>请求进程P（输入数字）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nEdit">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>1190</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>安全性检查</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>资源分配表</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>1225</y>
    </hint>
    <hint type="destinationlabel">
     <x>1049</x>
     <y>649</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
